### DISCUSSION TITLE
Laravel Vite & Tailwindcss JS = JS missing

### DISCUSSION THREAD

**Discussion by Radww** [tailwindlabs/tailwindcss#10623] [Help] [answered]
Hello there,

I tried to install tailwindui with Vite and using Laravel as backend. I followed the docs here: https://tailwindcss.com/docs/guides/laravel

But the only issue is that the JS part is missing at all. How can I make use of that? 😥

I took this example: https://tailwindui.com/components/marketing/elements/headers

But all what I can see is the page working (at css level) with both menus dropdowned: 
![image](https://user-images.githubusercontent.com/13894559/219799253-e95062a8-7983-44f7-a975-8b1286fbcdfd.png)

Sorry if this is a dumb question.

**Comment by wongjn** [comment, marked as answer]
If you are not using React or Vue, you would need to [roll your own JS](https://tailwindui.com/documentation#using-html-and-your-own-js).
